src/api/.rubocop_todo.yml

```diff
@@ -655,6 +655,7 @@ RSpec/AnyInstance:
     - 'spec/models/issue_tracker_spec.rb'
     - 'spec/models/package_spec.rb'
     - 'spec/models/project_spec.rb'
+    - 'spec/models/workflow/step/link_package_step_spec.rb'
```

Comment:
It's preferable to disable this cop only on the line where this applies.

Reply:
Done